### PR TITLE
Add open contest information

### PR DESCRIPTION
**Pull request by jonathanirvings**

### CODE REVIEW

File: open.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+### Technical Details
+
+To participate in APIO 2020 Open Contest, you will need to register for a
+[TLX](https://tlx.toki.id/) account if you do not have one. Once you are logged
+in to TLX, you can register the contest in
+[https://tlx.toki.id/contests/apio-2020-open](https://tlx.toki.id/contests/apio-2020-open).
+
+### Schedule
+
+The open contest will be held between Tuesday, 18 August 2020 (08:00 UTC+7) and
```
Comment (fushar) [inline open.md:18]: Kurang jelas, sepertinya perlu tambah seperti ini kali ya?

```
The open contest will be held between Tuesday, 18 August 2020 (08:00 UTC+7) and Thursday, 20 August 2020 (08:00 UTC+7). This is a virtual contest; you may start your participation any time within that window. Once the "Start" button ...

```

File: open.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+Thursday, 20 August 2020 (08:00 UTC+7). Once the "Start" button in the TLX
+contest is pressed, your individual 5-hour timer will be started.
+
+### Rules
+
+The rules of APIO 2020 Open Contest follows [the rules of APIO 2020](/rules),
```
Comment (fushar) [inline open.md:24]:
```suggestion
The rules of APIO 2020 Open Contest follow [the rules of APIO 2020](/rules),
```

File: open.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+key: open
+---
+
+We invite everybody to participate in APIO 2020 Open Contest.
+
+### Technical Details
+
+To participate in APIO 2020 Open Contest, you will need to register for a
+[TLX](https://tlx.toki.id/) account if you do not have one. Once you are logged
+in to TLX, you can register the contest in
```
Comment (fushar) [inline open.md:13]:
```suggestion
in to TLX, you can register the contest at
```

File: open.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+---
+layout: single-page-md
+title: Open Contest
+key: open
+---
+
+We invite everybody to participate in APIO 2020 Open Contest.
```
Comment (fushar) [inline open.md:7]: Tambah semacam: we politely ask contestants from APIO 2020 to not participate in this contest ... ?

File: open.md (hunk 2 above)
Comment (jonathanirvings) [reply open.md]: Done.

File: open.md (hunk 3 above)
Comment (jonathanirvings) [reply open.md]: Done.

File: open.md (hunk 1 above)
Comment (jonathanirvings) [reply open.md:18]: Done.

File: open.md (hunk 4 above)
Comment (jonathanirvings) [reply open.md:7]: Yang ini gw renungkan dulu :)

Comment (jonathanirvings) [reply open.md]: Added based on discussion on fb.

**Pull request merged**